railties/test/generators/app_generator_test.rb

```diff
@@ -1054,7 +1054,7 @@ def test_after_bundle_callback
       @bundle_commands_before_callback = @bundle_commands.dup
     end
 
-    run_generator_instance
+    run_generator_and_bundler [destination_root]
```

Comment:
Same comment here regarding `run_generator_and_bundler`.  See the call to `generator` on line 1053.  What is this fixing?